005-github-simplification.Rmd

```diff
@@ -0,0 +1,93 @@
+    This is particularly hard when you look holistically at all packages made by RStudio.
+
+-   There are a few packages that I am confused about:
+
+    -   Is rvest programming or data science? what about httr?
+    -   usethie is technically programming, it's very user facing, and we expect data scientists might use it for analysis projects.
+    -   glue is useful for data science but designed to be zero dep specifically for use by package developers.
+    -   Both tibble and magrritr are technically user-facing, but most people use via other packages. tibble is now really mostly about defining a data structure.
+    -   modelr is package that initially made sense in tidyverse, but is now mostly deprecated.
+    -   googledrive is not really about data analysis or package development.
+    -   vroom is in r-lib, readr is in tidyverse.
+
+-   As we start to use GitHub teams more, managing separate teams across separate orgs will become more painful.
+    It also makes it hard to centralise discussions, and to succinctly explain the scope of packages that tidyups apply to.
+
+-   It's painful to move issues across orgs.
+    This affects packages like dplyr where the underlying issue might need to be fixed in vctrs.
+
+## Solutions
+
+I think there are three possibilities:
+
+-   Move r-lib and tidymodels repos to **tidyverse org**.
+-   Move r-lib, tidyverse, and tidymodel repos to a **new org**.
+-   Move r-lib, tidyverse, and tidymodel to the **rstudio org**.
+
+These cases only affect GitHub organisations and in the short term all other websites would stay as is.
+This means that (e.g.) vroom would continue to live at vroom.r-lib.org.
+In the longer term, we'd probably make it vroom.{orgname}.org, but this would required some thought about how to manage the redirects, and whether it makes sense for (e.g.) parsnip to live at parsnip.{orgname}.org while the main tidymodels website continues to live at tidymodels.org.
+
+### Move to tidyverse org
+
+Pros:
+
+-   Moves smallest number of packages.
+-   Tidyverse has strongest brand identity.
+
+Cons:
+
+-   Makes it more complicated to explain exactly what the tidyverse is: it's a package, it's a set of packages loaded by `library(tidyverse)`, it's a set of packages installed by `install.packages("tidyverse")`, **and** it's a GitHub organisation.
+-   Binds all packages more closely to tidyverse. This might be less appealing for external developers and for other RStudio teams we might hope would also migrate to one org.
+-   No longer a one-to-one relationship between package homepage and GitHub organisation. (But could resolve in a second phase).
+-   Lose ability to bury less-important packages in r-lib; i.e. harder to highlight most important packages in tidyverse.
```

Comment:
Same point presented from the opposite direction: looks like there are an overwhelming number of packages in the tidyverse.